WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -360,6 +377,13 @@ extension RegisterDomainSuggestionsViewController {
         static let primaryButtonTitle = NSLocalizedString("Select domain",
                                                           comment: "Register domain - Title for the Choose domain button of Suggested domains screen")
         static let supportButtonTitle = NSLocalizedString("Help", comment: "Help button")
+
+        static let errorTitle = NSLocalizedString("domains.failure.title",
+                                                  value: "Sorry, the domain you are trying to add can not be bought on the Jetpack app at this time.",
```

Comment:
nitpick: "cannot" is [the more common spelling](https://www.thesaurus.com/e/grammar/cannot-or-can-not/) of "can not", although this is how WP.com put it.

Reply:
Yes, it looked strange to me too but I didn't want to dispute the WP.com translations. However, we can update it in the app to have a more common spelling.